web/views/package_view.ex

```diff
@@ -1,8 +1,12 @@
 defmodule HexWeb.PackageView do
   use HexWeb.Web, :view
 
+  def show_sort_info(nil), do: "(Sorted by name)"
   def show_sort_info(param) do
-    available_params = %{"name" => "name", "inserted_at" => "recently created", "downloads" => "downloads"}
+    available_params = %{
+      "name" => "name", "inserted_at" => "recently created",
+      "downloads" => "downloads", "updated_at" => "recently updated"}
+
```

Comment:
#319 And what about "recently updated" option
![2016-04-21 03 34 44](https://cloud.githubusercontent.com/assets/7054944/14686024/6898a0ea-077a-11e6-96c6-58459551fd09.png)
 in this selectbox?